exporter/metrics.go

```diff
@@ -88,7 +88,9 @@ func (e *Exporter) registerConstMetric(ch chan<- prometheus.Metric, metric strin
 	} else {
 		desc = e.mustFindMetricDescription(metric)
 	}
-
+	if e.options.AppendInstanceRoleLabel { // append instance_role label to all metrics
```

Comment:
If idea look ok in general, I'll try to add tests to cover it. 
Right now coverage decreased by 0.2%